docs/content/en/docs-dev/user-guide/adding-an-application.md

```diff
@@ -47,7 +42,7 @@ Click `ADD` to complete the registration.
 <p style="text-align: center;">
 </p>
 
-## From the web UI
+## Manually configuring application information
```

Comment:
```suggestion
## Configuring application information via register application form
```
nits

Reply:
I don't have a strong opinion on this but still prefer the original one.